src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MID_TERM = 3;
+  const MID_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+  const TOTAL_PRICE = days * PRICE_PER_DAY;
+
+  switch (true) {
+    case days >= LONG_TERM:
+      return TOTAL_PRICE - LONG_TERM_DISCOUNT;
+    case days >= MID_TERM:
+      return TOTAL_PRICE - MID_TERM_DISCOUNT;
```

Comment:
The logic for applying discounts is correct, but there is a potential issue: the discounts are fixed amounts, not percentages. This means that for very long rentals, the discount might not be significant compared to the total price. Ensure this is the intended behavior.